bin/update/00-run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
```

Comment:
```suggestion
#!/usr/bin/env bash
```

We shouldn't make assumptions about where `bash` exists. `/bin/bash` needs `bash` to exist at that path. `/usr/bin/env bash` will work as long as a `bash` executable exists somewhere in `$PATH`.

(This should be used on all scripts)